uint8 index
MissionWaypointTask waypoint_task
